src/app/submission/Submission.ts

```diff
@@ -63,7 +63,7 @@ export class Submission {
       throw Error('Payments are not handled by this class');
     }
     this.bsn = this.parsedSubmission.bsn;
-    this.kvk = this.parsedSubmission.kvk;
+    this.kvk = this.parsedSubmission.kvknummer;
```

Comment:
Even checken met @joostvanderborg of dit nu naar productie kan zo. Fix van de hackathon, maar dit betekent dat kvk submissions tonen op mijn-nijmegen nooit heeft gewerkt. Joost heeft gister een DB migratie/fix gedaan vgm.

Reply:
Kan nu dus idd naar prod.